test/users.js

```diff
@@ -0,0 +1,178 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  { User } = require('../app/models');
+
+chai.should();
+
+describe('/users POST', () => {
+  it('should pass sign up, parameters are valid', () => {
+    let response;
```

Comment:
there is no need for this let binding, just use the response

Reply:
I did this to have the response in the next promise. Otherwise I don't know how to "save" the value of a variable to use it in a future promise.
If I do:
```
       .then(res => {
        res.should.have.status(200);       
        return User.find({
          where: {
            firstName: 'Rodrigo',
            lastName: 'Aparicio',
            email: 'rodrigo.aparicio@wolox.com.ar'
          }
        });
      })
      .then(user => {
        console.log(res);
        dictum.chai(res, 'a new user is created with the parameters sent');
        // ERROR, res does not exist
      });
```
So how can I do that thing without using an auxiliar variable?